### PR TITLE
update map height, remove gis tools

### LINKED ISSUE #1205: update map height in search, also remove Remove gis tools and resources
Screenshot 2024-08-19 at 4 06 43 PM

### PR DESCRIPTION
**Pull request by dnoneill**
closes #1205

### CODE REVIEW

File: app/assets/stylesheets/earthworks.css [hunk 1]
```diff
@@ -103,4 +103,8 @@
 .sort-pagination, .pagination-search-widgets {
   padding-bottom: .5rem;
   border-bottom: 2px solid var(--bs-border-color);
+}
+
+#main-container .leaflet-container {
+  height: 620px;
```
Comment (thatbudakguy) [inline app/assets/stylesheets/earthworks.css:109]: is it still this big on mobile? seems pretty big

Comment (dnoneill) [reply app/assets/stylesheets/earthworks.css:109]: It looks like on mobile we are supposed to hide the map.

Comment (dnoneill) [reply app/assets/stylesheets/earthworks.css:109]: Updated the css to include mobile

File: app/assets/stylesheets/earthworks.css [hunk 2]
```diff
@@ -141,4 +141,14 @@ Uncomment lines 142-144
...
   display: none;
-} */
\ No newline at end of file
+} */
+
+#main-container .leaflet-container {
+  height: 620px;
+}
+
+@media (max-width: 575.98px) {
```
Comment (thatbudakguy) [inline app/assets/stylesheets/earthworks.css:150]: this is `--bs-breakpoint-sm`

Comment (thatbudakguy) [reply app/assets/stylesheets/earthworks.css:150]: might be a little more natural to invert it and use `min-width` of `--bs-breakpoint-md`

Comment (dnoneill) [reply app/assets/stylesheets/earthworks.css:150]: Thanks I was looking for those variables and couldn't find them anywhere.

Comment (dnoneill) [reply app/assets/stylesheets/earthworks.css:150]: I am trying @media (min-width: var(--bs-breakpoint-md)) {
  #main-container #leaflet-viewer  {
    display: none;
  }
}

and it isn't working. Not sure why

Comment (thatbudakguy) [reply app/assets/stylesheets/earthworks.css:150]: sigh. apparently `var()` doesn't work in media queries, but there are proposals to add it to CSS. nevermind

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**